catroid/src/org/catrobat/catroid/ui/MainMenuActivity.java

```diff
@@ -200,21 +199,9 @@ public void handleWebButton(View view) {
 	}
 
 	public void startWebViewActivity(String url) {
-		// TODO just a quick fix for not properly working webview on old devices
-		if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.GINGERBREAD_MR1) {
-			final SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(this);
-			boolean showBrowserWarning = preferences.getBoolean(SHARED_PREFERENCES_SHOW_BROWSER_WARNING, true);
-			if (showBrowserWarning) {
-				showWebWarningDialog();
```

Comment:
The method showWebWarningDialog() can be entirely removed, as it is not used anymore when deleting it here.